homeassistant/components/tuya/sensor.py

```diff
@@ -262,6 +262,29 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
             translation_key="last_amount",
             state_class=SensorStateClass.MEASUREMENT,
         ),
+        TuyaSensorEntityDescription(
+            key=DPCode.BATTERY_PERCENTAGE,
+            translation_key="battery",
+            native_unit_of_measurement=PERCENTAGE,
+            device_class=SensorDeviceClass.BATTERY,
+            state_class=SensorStateClass.MEASUREMENT,
+            entity_category=EntityCategory.DIAGNOSTIC,
+        ),
```

Comment:
We can use `*BATTERY_SENSORS` to handle all battery-based sensors in a single go.